deno.jsonc

```
@@ -0,0 +1,11 @@
+{
+    "lock": false,
+    "fmt": {
+        "proseWrap": "preserve",
+        "indentWidth": 4
```

```suggestion
        "indentWidth": 4,
        "lineWidth": 120
```

The default value of 80 doesn't make a lot of sense with modern screens being easily able to fit more, and it leads to horrible looking lines that get broken in the middle with absolutely no need

---

I often have two source files open next to each other and 80 characters is a sane limit for me.